src/main/java/org/bytedeco/javacpp/Loader.java

```diff
@@ -1276,38 +1273,12 @@ public static String load(Class cls, Properties properties, boolean pathsFirst)
                         }
                     }
                 }
-                for (String executable : executables) {
-                    String[] split = executable.split("#");
-                    filename = prefix + split[0] + suffix;
-                    String filename2 = split.length > 1 ? prefix + split[1] + suffix : null;
-                    for (int i = extensions.length - 1; i >= -1; i--) {
-                        // iterate extensions in reverse to be consistent with the overriding of properties
-                        String extension = i >= 0 ? extensions[i] : "";
-                        String subdir = libraryPath.length() > 0 ? "/" + libraryPath : platform + (extension == null ? "" : extension);
-                        URL u = findResource(cls, subdir + "/" + filename);
-                        if (u != null) {
-                            if (filename2 != null) {
-                                u = new URL(u + "#" + filename2);
-                                if (!u.toString().contains("#")) {
-                                    Field f = URL.class.getDeclaredField("ref");
-                                    f.setAccessible(true);
-                                    f.set(u, filename2);
-                                }
-                            }
-                            File f = cacheResource(u);
-                            if (f != null) {
-                                f.setExecutable(true);
-                                executablePaths.add(f.getAbsolutePath());
-                            }
-                        }
-                    }
-                }
-            } catch (IOException | NoSuchFieldException | IllegalAccessException e) {
+            } catch (IOException e) {
                 logger.error("Could not extract executable " + filename + ": " + e);
             }
         }
         if (executables.size() > 0) {
-            return executablePaths.size() > 0 ? executablePaths.get(0) : null;
+            return executablePaths.size() > 0 ? executablePaths.get(executables.get(0)) : null;
```

Comment:
Now that executablePaths is a `Map<String, String>` we can get the old behavior of returning the first executable by passing the first executable name.